locale/po/datastore_search.po

```diff
@@ -563,91 +449,119 @@ msgid ""
-"        "
 
 # advanced
 msgid "Advanced"
-msgstr "Avancée"
+msgstr ""
 
 # found_activities
 msgid "Found <b>%{count}</b> activities"
-msgstr "<b>%{count}</b> activités trouvées"
+msgstr ""
 
 # relevance
 msgid "Relevance"
-msgstr "Pertinence"
+msgstr ""
+
+# sort_relevance_label
+msgid "Sort results by relevance"
+msgstr ""
 
 # identifier
 msgid "Identifier"
-msgstr "Identifiant"
+msgstr ""
+
+# sort_identifier_label
+msgid "Sort results by IATI identifier"
+msgstr ""
 
 # all_narratives
 msgid "All Narratives"
-msgstr "Tout le texte"
+msgstr ""
 
 # all_narratives_desc
 msgid "Searches all IATI narrative fields, used by simple search"
-msgstr "Recherche tous les champs narratifs de l’IITA, utilisés par une recherche simple"
+msgstr ""
 
 # boolean_grouping
 msgid "Boolean Grouping"
-msgstr "Regroupement Booléen"
+msgstr ""
 
 # boolean_grouping_desc
 msgid "Parenthesis for grouping boolean queries. Ensure every opening parenthesis is matched with a closing parenthesis."
-msgstr "Parenthèse pour regrouper les requêtes booléennes. Assurez-vous que chaque parenthèse ouvrante correspond à une parenthèse fermante."
+msgstr ""
 
 # special_fields
 msgid "Special fields"
-msgstr "Champs spéciaux"
+msgstr ""
 
 # grouping
 msgid "Grouping"
-msgstr "Regroupement"
+msgstr ""
 
 # standard_fields
 msgid "Standard fields"
-msgstr "Champs standards"
+msgstr ""
 
 # incompatible_file_error
 msgid "Incompatible file detected. Please try a different file."
-msgstr "Fichier incompatible détecté. Veuillez essayer un autre fichier."
+msgstr ""
 
 # search_term_is_required
 msgid "Search term is required"
-msgstr "Le terme de recherche est requis"
+msgstr ""
 
 # selection_is_required
 msgid "Selection is required"
-msgstr "Une sélection est requise"
+msgstr ""
 
 # value_is_required
 msgid "Value is required"
-msgstr "Valeur est requise"
+msgstr ""
 
 # date_is_required
 msgid "Date is required"
-msgstr "La date est requise"
+msgstr ""
 
 # is_not_allowed
 msgid "%{bad} is not allowed for Datastore Search queries"
-msgstr "%{bad} n’est pas autorisé pour les requêtes Rechercher le Datastore"
+msgstr ""
 
 # percentage_validation
 msgid "Percentage must be between 0 and 100"
-msgstr "Le pourcentage doit être compris entre 0 et 100"
+msgstr ""
 
 # integer_validation
 msgid "Value must be a whole number"
-msgstr "La valeur doit être un nombre entier"
+msgstr ""
 
 # fetch_error
 msgid "There was an error fetching your query. Please check how your query is constructed and try again."
-msgstr "Une erreur s’est produite lors de la récupération de votre requête. Veuillez vérifier la structure de votre requête et réessayer."
+msgstr ""
 
 # search_term
 msgid "Search term"
-msgstr "Le terme de recherche"
+msgstr ""
+
+# spatial_search
+msgid "Geospatial search"
+msgstr ""
+
+# location_point_latlon_desc
+msgid "Spatial search on location/point/pos latitude and longitude"
+msgstr ""
+
+# latlon_placeholder
```

Comment:
Not sure this makes sense to translate?

Reply:
I mean the `latlon_placeholder` one, comment highlighted a different part than I expected.

Reply:
There will be a few snippets of text we don't end up using. When discussing with UNDP we decided that since most of the dropdown fields aren't going to be translated, it didn't make sense to translate the rest (e.g. "All narratives"). But I figure it's worth having them translated and available, and then we can cut them out and replace with static copy after receiving the PO file back.

Reply:
Sounds good. I guess the estimated cost is not that much per word, so this isn't going to break the budget.